core/src/components/divider/divider.ionic.scss

```
@@ -0,0 +1,47 @@
+@use "../../themes/ionic/ionic.globals.scss" as globals;
+@import "./divider.common.scss";
```

We don't want to use `@import` anymore because it's deprecated. We only do it with the ionic theme. At some point, we'll make the shift for iOS and md to no longer use `@import`.

```suggestion
@use "./divider.common.scss";
```